ansible/inventories/ci/group_vars/dashboard-web/vars.yml

```diff
@@ -1,3 +1,27 @@
 ---
 datagov_service: dashboard
-cron_to_emails: "{{ datagov_team_email }}"
+app_base_url: https://dashboard.ci.datagov.us
+default_host: dashboard.ci.datagov.us
+
+# MySQL credentials
+env_db_host: "{{ vault_env_db_host }}"
+env_db_user: "{{ vault_env_db_user }}"
+env_db_pass: "{{ vault_env_db_pass }}"
+env_db_name: "{{ vault_env_db_name }}"
+
+# AWS S3 bucket
+s3_bucket: bsp-ocsit-dev-east-appdata
```

Comment:
Can you open an issue for this and add a comment? We need to create an S3 bucket as part of terraform and use that bucket here. This bucket won't work for the sandbox environments.

Reply:
Sure, will do.

Reply:
I'm going to do this in the course of resolving https://github.com/GSA/datagov-deploy/issues/1152, which I need to do before this gets merged in any case!